tests/integration/validate/test_validate_command.py

```diff
@@ -59,10 +65,11 @@ def command_list(
             ),  # project with template.json and standard build directory .aws-sam/build/template.yaml
         ]
     )
-    def test_default_template(self, relative_folder: str, expected_file: TemplateFileTypes):
-        cwd = f"tests/integration/testdata/validate/{relative_folder}"
-        command_result = run_command(self.command_list(), cwd=cwd)
+    def test_default_template_file_choice(self, relative_folder: str, expected_file: TemplateFileTypes):
+        test_data_path = Path(__file__).resolve().parents[2] / "integration" / "testdata" / "validate"
```

Comment:
I didn't know `/` operator can be used in `Path`, wow!

Reply:
yeah, it's nice to have operator overloading 😄

Reply:
is the "/" similar to using pathlib.join ?

Reply:
yeah I tested, it is